ckanext/who_afro/templates/snippets/dev_banner.html

```diff
@@ -0,0 +1,5 @@
+{% if under_development == True %}
+    <div class="account-masthead dev-banner-mobile">
+        <a href="#"><img src="/images/alert-circle-blue.png" alt="alert"/>BETA - Site under development<span class="clickable">Learn more</span></a>
+    </div>
```

Comment:
Can this be pre-pended to language_selector.html to keep it all in one file? Allowing us to delete this file?

Reply:
I have deleted the said file.